sdk/CHANGELOG.md

```diff
@@ -110,6 +111,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Rename `Account::prepare_mint_native_token` to `prepare_create_native_token`, `Account::prepare_increase_native_token_supply` to `prepare_mint_native_token`, `Account::prepare_decrease_native_token_supply` to `prepare_melt_native_token`;
 - Rename `MintNativeTokenParams` to `CreateNativeTokenParams`;
 - Rename `MintNativeTokenTransaction` to `CreateNativeTokenTransaction` and `PreparedMintNativeTokenTransaction` to `PreparedCreateNativeTokenTransaction` (including their corresponding DTOs);
+- Made `ManagerStorage` public and renamed it to `StorageKind`;
+- Fields of `StorageOptions` are now public;
```

Comment:
They are not?

Reply:
Still not public :trollface:

Reply:
Yeah yeah I haven't done it yet, chill :trollface:

Reply:
Done